src/grayscale-theme/index.ts

```diff
@@ -0,0 +1 @@
+export { GrayscaleThemeProvider } from './GrayscaleThemeProvider'; 
```

Comment:
Always insert a final newline on files so this doesn't show up. In VSCode, this can be set `files.insertFinalNewline: true`

![image](https://github.com/user-attachments/assets/4a7d233e-8436-45b9-9d05-e8e054a4cb07)